src/btwa_api/app/models/handler.py

```diff
@@ -0,0 +1,9 @@
+from sqlalchemy import Binary, Column, Integer, String
+
+from ..db.base_class import Base
+
+
+class Handler(Base):
+    id = Column(Integer, primary_key=True, index=True)
+    username = Column(String(length=128), unique=True)
+    password = Column(Binary(length=80))
```

Comment:
I would suggest to store password as UTF-8 string, mostly because it is more common. I realize that bcrypt is returning bytes object in py3 , but I think it's just for purpose that everyone can choose any storage encoding then.

Reply:
Is bcrypt output UTF-8 safe?

Reply:
I do think so

```
>>> import bcrypt
>>> password = b"somepassword"
>>> hashed = bcrypt.hashpw(password, bcrypt.gensalt())
>>> hashed
b'$2b$12$fWyB1OeA6/K6djQ7eWDHwey.diis0hDYJTX4Z8vOT.VxjZ5rWmpsm'
```

Reply:
based on slack discussion, there no strong reason against binary storage except common practice in other languages and projects. decided to not change the impl.